CHANGELOG.md

```diff
@@ -382,6 +382,7 @@
 - Policy markers added to activity form, including BEIS custom answer `not assessed`
 - Add a field to report whether the activity has any relation to the Covid-19 pandemic.
 - Include the Covid-19 field in the CSV report.
+- Append the text "COVID-19" to the activity description in the IATI XML export, where applicable.
```

Comment:
FYI - I usually try to stick to one CHANGELOG entry per PR.

Reply:
I was told that the changelog entry should belong to the most relevant commit, and I like small commits, they're much easier to review than one giant pull request with a hundred changed files. This sort of leads into there being a few "main" commits where the spirit of the "thing being changed" is split between. 🤷🏻‍♀️